lib/components/form/element/combobox.js

```diff
@@ -835,11 +853,13 @@ class Combobox extends React.Component {
         }
         matches = Array.from(matches);
 
-        const options = _(matches).map((option) => ({
+        const filterOption = (option) => ({
```

Comment:
```suggestion
        const formatOption = (option) => ({
```

We're not filtering options here, we just formatting them, right?

Reply:
Yes, I've modified the naming to reflect that.